packages/components/src/components/post-breadcrumb-item/post-breadcrumb-item.tsx

```diff
@@ -0,0 +1,49 @@
+import { Component, Element, h, Host, Prop, Watch } from '@stencil/core';
+import { version } from '@root/package.json';
+import { checkUrl } from '@/utils';
+
+/**
+ * @slot default - Slot for placing the text inside the breadcrumb item.
+ */
+@Component({
+  tag: 'post-breadcrumb-item',
+  styleUrl: 'post-breadcrumb-item.scss',
+  shadow: true,
+})
+export class PostBreadcrumbItem {
+  @Element() host: HTMLPostBreadcrumbItemElement;
+
+  /**
+   * The optional URL to which the breadcrumb item will link.
+   */
+  @Prop() url?: string | URL;
+
+  @Watch('url')
+  validateUrl() {
+    checkUrl(this.url, 'The "url" property of the post-breadcrumb-item is invalid');
```

Comment:
Since the URL is optional, you should use an `emptyOr` validator like so:
```suggestion
    checkEmptyOrUrl(this.url, 'The "url" property of the post-breadcrumb-item is invalid');
```